### PR TITLE
view pdf

### PR DESCRIPTION
**Pull request by cristianarce89**

### CODE REVIEW

**Review by Daniielmayo — APPROVED** [review]

File: components/organisms/Viewpdf/Viewpdf.module.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    width: 80%;
+    height: 29.375rem;
+}
+
+.pdf {
+    width: 100%;
+    height: 100%;
+}
+
+@media (width >=48rem) {
```
Comment (SthefaniaEcheverry) [inline components/organisms/Viewpdf/Viewpdf.module.css:54]: Add: 
```
    .viewpdf-container {
        padding-left: 6rem;
    }
```

Comment (SthefaniaEcheverry) [inline components/organisms/Viewpdf/Viewpdf.module.css:54]: From this measurement, the PDF should go to the end of the view

This is how it is:
image

This is how it should be:
image

**Review by SthefaniaEcheverry — CHANGES_REQUESTED** [review]
I made some comments

File: components/templates/Viewpdf-template/ViewpdfTemplate.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { Viewpdf } from "../../organisms";
+
+export const ViewpdfTemplate: React.FC = () => {
+    return (        
+        <div>
```
Comment (Daniielmayo) [inline components/templates/Viewpdf-template/ViewpdfTemplate.tsx:5]: removes the div and only leaves the component

File: src/app/(privateRoutes)/dashboard/viewpdf/page.tsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React from 'react'
```
Comment (Daniielmayo) [inline src/app/(privateRoutes)/dashboard/viewpdf/page.tsx:1]: Is this import necessary?
if it is not eliminated

**Review by Daniielmayo — CHANGES_REQUESTED** [review]
I made some comments

File: components/organisms/Viewpdf/Viewpdf.module.css [hunk 4]
```diff
@@ -0,0 +1,98 @@
+.viewpdf-container {
```
[the remark this answers is not in the document]
Comment (cristianarce89) [reply components/organisms/Viewpdf/Viewpdf.module.css:1]: add

File: components/organisms/Viewpdf/Viewpdf.module.css [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+    height: 100vh;
+    width: 100vw;
+    background-image: url(https://i.imgur.com/UQqp7LT.png);
+    background-size: 100% 100%;
+    overflow-y: scroll;
+}
+
+.viewpdf-containerBackButton {
+    display: flex;
+    position: absolute;
```
[the remark this answers is not in the document]
Comment (cristianarce89) [reply components/organisms/Viewpdf/Viewpdf.module.css:12]: .viewpdf-containerBackButton { 
este debe llevar position: absolute ya que sino se corre completamente

el color color: var(--darkBlue) se lo puse al .viewpdf-backButton quedo asi:

.viewpdf-containerBackButton {
    display: flex;
    position: absolute;
    font-size: var(--fontSize-titles_mobile);
    z-index: 1;
    top: 0.313rem;
}

.viewpdf-backButton {
    display: flex;
    align-items: center;
    gap: 0.5rem;
    text-decoration: none;
    font-size: var(--fontSize-buttons_mobile);
    transition: var(--transition);
    background-color: transparent;
    border: none;
    cursor: pointer;
    color: var(--darkBlue)
}

File: components/organisms/Viewpdf/Viewpdf.module.css (hunk 1 above)
Comment (cristianarce89) [reply components/organisms/Viewpdf/Viewpdf.module.css:54]: responsive bad

Comment (cristianarce89) [reply components/organisms/Viewpdf/Viewpdf.module.css:54]: image
so ?

File: src/app/(privateRoutes)/dashboard/viewpdf/page.tsx (hunk 3 above)
Comment (cristianarce89) [reply src/app/(privateRoutes)/dashboard/viewpdf/page.tsx:1]: remove

**Review by SthefaniaEcheverry — APPROVED** [review]

**Review by Daniielmayo — APPROVED** [review]

**Review by Daniielmayo — APPROVED** [review]

**Review by SthefaniaEcheverry — APPROVED** [review]

**Pull request merged**